.env.example

```diff
@@ -114,6 +114,9 @@ LOG_LEVEL=
 # NOTE: you need to restart the openCloud container to load the new extensions.
 # OC_APPS_DIR=/your/local/opencloud/apps
 
+# If you're using an external IdP and don't want to use standard docker volumes for ldap-server, you can define local paths here
```

Comment:
The phrase `you can define/set [...] here`

Doesn’t align with the tone in the .env file. 
You might want to rephrase it to better match the tone used elsewhere.